_projects/10-scala-native.md

```diff
@@ -8,7 +8,7 @@ contributors: [martinduhem, guillaume, ergys]
 status: Active
```

Comment:
I just noticed that we should add @errikos to the contributors list

Reply:
(and after #140 is done we should add Wojciech)

Reply:
looks like Ergys is already there